File: SDKs/keycloak.local/1/python/src/sdk/models/operations/get_realm_groups_id_role_mappings_clients_client_available.py
from dataclasses import dataclass, field
from typing import List,Optional
from sdk.models import shared


@dataclass
class GetRealmGroupsIDRoleMappingsClientsClientAvailablePathParams:
    client: str = field(default=None, metadata={'path_param': { 'field_name': 'client', 'style': 'simple', 'explode': False }})
    id: str = field(default=None, metadata={'path_param': { 'field_name': 'id', 'style': 'simple', 'explode': False }})
    realm: str = field(default=None, metadata={'path_param': { 'field_name': 'realm', 'style': 'simple', 'explode': False }})
    

@dataclass
class GetRealmGroupsIDRoleMappingsClientsClientAvailableRequest:
    path_params: GetRealmGroupsIDRoleMappingsClientsClientAvailablePathParams = field(default=None)
    

@dataclass
class GetRealmGroupsIDRoleMappingsClientsClientAvailableResponse:
    content_type: str = field(default=None)
    role_representations: Optional[List[shared.RoleRepresentation]] = field(default=None)
    status_code: int = field(default=None)
    

File: SDKs/keycloak.local/1/go/pkg/models/operations/getrealmgroupsidrolemappingsclientsclientavailable.go
package operations

import (
"openapi/pkg/models/shared")

type GetRealmGroupsIDRoleMappingsClientsClientAvailablePathParams struct {
    Client string `pathParam:"style=simple,explode=false,name=client"`
    ID string `pathParam:"style=simple,explode=false,name=id"`
    Realm string `pathParam:"style=simple,explode=false,name=realm"`
    
}

type GetRealmGroupsIDRoleMappingsClientsClientAvailableRequest struct {
    PathParams GetRealmGroupsIDRoleMappingsClientsClientAvailablePathParams 
    
}

type GetRealmGroupsIDRoleMappingsClientsClientAvailableResponse struct {
    ContentType string 
    RoleRepresentations []shared.RoleRepresentation 
    StatusCode int64 
    
}

